src/components/VariationsMainView/VariationsMainView.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Variations Main View 
+menu: Components
```

Comment:
Could we put this under proofs of concepts, or maybe replace the material design view, its very similar.

Reply:
done https://github.com/digirati-co-uk/timeliner/pull/67/commits/294f139f3297c19c0f215a4244c752e81a36f24a